DFXR3Editor/Defs/DefActionID.xml

```diff
@@ -137,34 +269,34 @@
 				<field name="" dataType="f32"/>
 			</Fields2>
 		</ActionID>
-		<ActionID ID="601" name="Line(Color)">
+		<ActionID ID="601" name="Line">
```

Comment:
I agree with this one, as there's not a line that doesn't take a color argument. However as said before the function style syntax does currently serve a purpose and shouldn't be removed unless you find a better visual cue.